app/src/main/java/pl/ahyoshin/demo/entities/AuthenticationEntity.java

```diff
@@ -0,0 +1,38 @@
+package pl.ahyoshin.demo.entities;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "authorities")
+public class AuthenticationEntity {
+    @Id
+    @Column(name = "authorities_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String authority;
+
+    public AuthenticationEntity() {
+    }
+
+    public Long getId() {
+        return this.id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getAuthority() {
+        return this.authority;
+    }
+
+    public void setAuthority(String authority) {
+        this.authority = authority;
+    }
+}
```

Comment:
Nie wiem czy jest nam potrzebna ta klasa, bo adnotacje w UserEntity stworzą tę tabelę anyway + to chyba generuje sqlowe wyjątki

Reply:
nie mam pewnosci ale jak czytalem o tworzeniu tabel to pokazywali ze trzeba tworzyc